### DISCUSSION TITLE
New interface proposal for the Checklists

### DISCUSSION THREAD

**Discussion by andergmartins** [publishpress/publishpress-checklists#433] [Feature Requests]
### Arguments

In the past we had users complaining about the way it looks when you have many custom post types, so we fixed adding multiple lines of buttons:

<img width="1279" alt="Screen Shot 2020-07-27 at 14 15 39" src="https://user-images.githubusercontent.com/407131/88571218-aa662b80-d013-11ea-9e35-c759d9b0092f.png">

Other users are suggesting customize the warning message: #171 
This could be done adding a tab for Tasks, and another for Messages, or other configuration we could have:

<img width="1278" alt="Screen Shot 2020-07-27 at 14 19 16" src="https://user-images.githubusercontent.com/407131/88571553-324c3580-d014-11ea-9bc7-e0db2acb2849.png">

But those similar UI elements look confuse together. 

Another argument is that if we want to configure the checklist for a custom type that is represented by the last button there, we see the Posts is always loaded, since it is the first option there.

Yet another argument, is that we plan to have multiple checklists, #100 and my following proposal is a intermediary step for achieving that.

### Proposal
So I would like to propose we have a separate page for a list of checklists. Right now, one for each post type by default, with a column that says how many tasks we have enabled for the checklist:

<img width="1282" alt="Screen Shot 2020-07-27 at 15 10 31" src="https://user-images.githubusercontent.com/407131/88576133-565f4500-d01b-11ea-9fd9-c35b672f5e83.png">

Then clicking on the post type, we get a screen like the mockup added to the task #172 
Btw, I suggest using the WP's native table class WP_List_Table for now.

**Comment by stevejburge** [comment]
Great idea, @andergmartins That would give us more freedom.

This main screen could also have extra details such as whether or not the checklist is enabled.

Also thinking ahead, we hope to have different checklists for different statuses. The status info could show here too.
